Source/items.h

```diff
@@ -197,7 +197,7 @@ struct Item {
 	uint8_t _iMinDam = 0;
 	uint8_t _iMaxDam = 0;
 	int16_t _iAC = 0;
-	uint32_t _iFlags = 0; // item_special_effect
+	ItemSpecialEffect _iFlags = ItemSpecialEffect::None; // ItemSpecialEffect flags

```

Comment:
```suggestion
	ItemSpecialEffect _iFlags = ItemSpecialEffect::None;
```
This comment doesn't add any value now that you've given the member a proper type :)